subsys/net/lib/dns/mdns_responder.c

```diff
@@ -691,4 +712,16 @@ static int mdns_responder_init(void)
 	return init_listener();
 }
 
+int mdns_responder_set_ext_records(struct dns_sd_rec records[], size_t count)
```

Comment:
Please use a (const) pointer and size rather than the array and size.
```suggestion
int mdns_responder_set_ext_records(const struct dns_sd_rec *records, size_t count)
```